pkg/metrics/metrics.go

```diff
@@ -73,6 +73,11 @@ const (
 	// ClusterStatusCapacityAvailable - metric name for the number of available instances
 	ClusterStatusCapacityAvailable = "cluster_status_capacity_available"
 
+	// KasFleetManagerClusterProviderResourceQuotaConsumed - metric name for how much quota an organisation is currently using
+	KasFleetManagerClusterProviderResourceQuotaConsumed = "kas_fleet_manager_cluster_provider_resource_quota_consumed"
```

Comment:
No need to add KasFleetManager as a prefix. This is taken care of by `Subsystem: KasFleetManager,` in the metric definition. 

```suggestion
	// KasFleetManagerClusterProviderResourceQuotaConsumed - metric name for how much quota an organisation is currently using
	ClusterProviderResourceQuotaConsumed = "cluster_provider_resource_quota_consumed"
```